lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -136,6 +136,13 @@ class ConfigSchemaHandler {
             this.serverless.cli.log(`  ${message}`, 'Serverless', { color: 'orange' });
           }
         }
+        if (!this.serverless.configurationInput.configValidationMode) {
+          this.serverless._logDeprecation(
+            'CONFIG_VALIDATION_MODE_DEFAULT',
+            'Starting with next major, Serverless will throw on configuration errors by default....\n' +
```

Comment:
Sorry! When I suggested "Starting with next major, Serverless will throw on configuration errors by default..." by three dots I meant to just continue with the rest of the message - sorry for not being clear. Let's just end with a dot instead of multiple dots. I think we can also remove the newline character and display the 'Adapt (...)' sentence right after the first one.